test/components/popup.spec.tsx

```diff
@@ -60,8 +60,8 @@ describe('<Popup />', () => {
         ).withDriver(PopupTestDriver);
 
         return waitFor(() => {
-            expect(popup.root).to.be.present();
-            expect(popup.content[0]).to.be.present();
+            expect(popup.root).to.not.null;
+            expect(popup.content[0]).to.not.null;
         });
```

Comment:
what is the difference?

Reply:
there are no difference. I have beed playing with tests and forgot to cleanup. Now I restored original tests